hypothesis-python/src/hypothesis/strategies/_internal/collections.py

```diff
@@ -286,8 +286,8 @@ def do_draw(self, data: ConjectureData) -> list[Ex]:
         def not_yet_in_unique_list(val: Ex) -> bool:  # type: ignore # covariant type param
             return all(key(val) not in seen for key, seen in zip(self.keys, seen_sets))
 
-        filtered = self.element_strategy._filter_for_filtered_draw(
-            not_yet_in_unique_list
+        filtered = FilteredStrategy(
+            self.element_strategy, conditions=(not_yet_in_unique_list,)
```

Comment:
I don't really understand the purpose of  `_filter_for_filtered_draw`, because it was never overridden. It seems to just be a way to write `FilteredStrategy(s, [condition])` inline?